src/Raven.Server/RavenServer.cs

```diff
@@ -1240,12 +1246,20 @@ public async Task StartCertificateReplicationAsync(byte[] certBytes, bool replac
                     throw new InvalidOperationException("Failed to load (and validate) the new certificate which was received during the refresh process.", e);
                 }
 
+                if (Certificate.Certificate.Thumbprint == newCertificate.Thumbprint)
+                {
+                    if (Logger.IsOperationsEnabled)
+                    {
+                        Logger.Operations($"The new certificate matches the current one. No further steps needed. {Certificate.Certificate.GetBasicCertificateInfo()}");
+                    }
+                    return;
+                }
+                
                 if (Logger.IsOperationsEnabled)
                 {
-                    var source = string.IsNullOrEmpty(Configuration.Security.CertificateLoadExec) ? "Let's Encrypt" : $"executable ({Configuration.Security.CertificateLoadExec} {Configuration.Security.CertificateLoadExecArguments})";
-                    Logger.Operations($"Got new certificate from {source}. Starting certificate replication.");
+                    Logger.Operations($"Starting certificate replication. current:\"{Certificate.Certificate.GetBasicCertificateInfo()}\", new:\"{newCertificate.GetBasicCertificateInfo()}\"");
```

Comment:
wrong casing, we are using ' instead of " everywhere else

Reply:
done